Makefile

```diff
@@ -0,0 +1,23 @@
+default: expressjs quarkus
+
+frontend:
+	+$(MAKE) -C frontend
+
+frontend.clean:
+	+$(MAKE) -C frontend clean
+
+expressjs:
+	+$(MAKE) -C expressjs
+
+expressjs.clean:
+	+$(MAKE) -C expressjs clean
+
+quarkus:
+	+$(MAKE) -C quarkus
+
+quarkus.clean:
+	+$(MAKE) -C quarkus clean
+
+clean: frontend.clean expressjs.clean quarkus.clean
```

Comment:
Is the showcase application supposed to be used as Knative service? The concern is: when the user connects to the stream of events, and the Ksvc possibly scales up and receives some events, will the client see all events ? Or will they miss some events because they will be sent to some other Pod of the knative service. Just curious.

Reply:
> Those files are there, so you can easily showcase the ease of use of the Ksvc as compared to the lengthy K8s/OCP way: openshift.yaml and plain.yaml. They reference the already published images, so to works out-of-the-box. I had used those exactly like that - showing people how they manifest differ, and how to apply on live demo.

OK. Then it makes sense to move the files from ROOT/quarkus/src/main/kubernetes/ somewhere higher (e.g. ROOT/kubernetes) and describe their purpose in the readme. Right now it looks like they're part of the built quarkus application because they're under src/main.

Reply:
I don't remember, exactly, why I put them in that dir. It may be in connection to the Quarkus Kubernetes extension default search paths.

But, this isn't done in this PR. Let's create an issue to track this.

Reply:
Tracked in https://github.com/openshift-knative/showcase/issues/14

Reply:
Also, here: https://github.com/openshift-knative/showcase/issues/17